src/BasisTheory.net.Tests/Tokenize/TokenizeTests.cs

```diff
@@ -90,7 +90,7 @@ public async Task ShouldCreateWithPerRequestApiKey(Func<ITokenizeClient, dynamic
 
     [Theory]
     [MemberData(nameof(Methods))]
-    public async Task ShouldCreateWithCorrelationId(Func<ITokenizeClient, dynamic, RequestOptions, Task<JToken>> mut)
+    public async Task ShouldCreateWithCustomHeaders(Func<ITokenizeClient, dynamic, RequestOptions, Task<JToken>> mut)
```

Comment:
The test changes are missing in this file to assert on the new header

Reply:
nice catch